src/components/Footer.js

```diff
@@ -1,17 +1,21 @@
 import data from "@/config/app.json";
+import { FaGithub } from "react-icons/fa";
 
 export default function Footer() {
   return (
-    <footer>
-      <div className="mx-auto max-w-3xl px-4 sm:px-6 lg:max-w-7xl lg:px-8">
-        <div className="border-t border-gray-200 py-8 text-center text-sm text-gray-500 sm:text-left">
-          <span className="block sm:inline">
-            &copy; {new Date().getFullYear()} EddieHub
-          </span>{" "}
-          <span className="block sm:inline">
-            All rights reserved | v{data.version}
-          </span>
+    <footer className="border-t border-gray-200 py-8">
+      <div className="mx-auto max-w-3xl px-4 sm:px-6 lg:max-w-7xl lg:px-8 flex justify-between items-center">
+        <div className="text-sm text-gray-500">
+          &copy; {new Date().getFullYear()} EddieHub All rights reserved | v{data.version}
         </div>
+        <a
+          href="https://github.com/EddieHubCommunity/CreatorsRegistry"
+          className="text-gray-500 hover:text-gray-600"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <FaGithub className="h-6 w-6 inline-block mb-1" />
```

Comment:
You can use the font awesome icon

```js
<FontAwesomeIcon
   icon={socialIcon("github")}
   className="h-6 w-6"
   style={{ height: 25, width: 25 }}
/>
```

But as we have the github logo in the header, i think in the footer show the full github url https://github.com/EddieHubCommunity/CreatorsRegistry

Reply:
So, instead of the icon, do you want like this? 
![image](https://github.com/EddieHubCommunity/CreatorsRegistry/assets/79015420/4fe6299c-13a7-41c5-8651-6feddb95fa2f)

Reply:
yes 👍 looks good